meteorite_fall.js

```diff
@@ -19,14 +20,18 @@ function main() {
 // main game functionality
 function loopGame() {
     let randomMeteoriteIndex = Math.floor(Math.random() * meteorites.length);
-
+    isGameOver = false;
     context.clearRect(0,0, canvasWidth, canvasHeight);
 
     if (meteorites[randomMeteoriteIndex].isFalling == false) {
         meteorites[randomMeteoriteIndex].isFalling = true;
     }
 
     for (arrayIndex = 0; arrayIndex < meteorites.length; arrayIndex++) {
+        checkGameOver(meteorites[arrayIndex].x, meteorites[arrayIndex].y, player.x, player.y, meteorites[arrayIndex].width, meteorites[arrayIndex].height, player.width);
```

Comment:
Instead of passing all these params, why not just pass the meteorite and player objects themselves? I think this would be cleaner.